File: Python/URI/1048.py
salario = float(input())


if(salario >= 0 and salario <= 400.00):
    
        reajuste = salario * 0.15
        salario = salario + reajuste
        percentual = 15
    
elif (salario >= 400.01 and salario <= 800.00):
    
        reajuste = salario * 0.12
        salario = salario + reajuste
        percentual = 12
    
elif(salario >= 800.01 and salario <= 1200.00):
    
        reajuste = salario * 0.10
        salario = salario + reajuste
        percentual = 10
elif (salario >= 1200.01 and salario <= 2000.00):
    
        reajuste = salario * 0.07
        salario = salario + reajuste
        percentual = 7
    
elif (salario > 2000.00):
    
        reajuste = salario * 0.04
        salario = salario + reajuste
        percentual = 4

print("Novo salario: {:.2f}".format(salario))
print("Reajuste ganho: {:.2f}".format(reajuste))
print("Em percentual: "+str(percentual)+" %")  

File: JS/URI/1048.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');



var salario = parseFloat(lines.shift());
var reajuste = 0;
var percentual = 0;

if(salario >= 0 && salario <= 400.00)
    {
        reajuste = salario * 0.15;
        salario = salario + reajuste;
        percentual = 15;
    }
else if (salario >= 400.01 && salario <= 800.00)
    {
        reajuste = salario * 0.12;
        salario = salario + reajuste;
        percentual = 12;
    }
else if (salario >= 800.01 && salario <= 1200.00)
    {
        reajuste = salario * 0.10;
        salario = salario + reajuste; 
        percentual = 10;
    }
else if (salario >= 1200.01 && salario <= 2000.00)
    {
        reajuste = salario * 0.07;
        salario = salario + reajuste;
        percentual = 7;
    }
else if (salario > 2000.00)
    {
        reajuste = salario * 0.04;
        salario = salario + reajuste;
        percentual = 4;
    }

console.log("Novo salario: "+salario.toFixed(2));
console.log("Reajuste ganho: "+reajuste.toFixed(2));
console.log("Em percentual: "+percentual+" %");

